includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -2,6 +2,8 @@
 
 class DesignSystemGlobalFooterModel extends WikiaModel {
 	const DEFAULT_LANG = 'en';
+	const WIKI_PRODUCT = 'wikis';
+	const FANDOM_PRODUCT = 'fandoms';
```

Comment:
https://github.com/Wikia/chef-repo/pull/11077/files#diff-d79267443db62896f1c9945fa9a26101R167 uses 'product=fandom' - this needs to be unified, for consistency I recommend to use plural in https://github.com/Wikia/chef-repo/pull/11077